frontend/public/src/components/CourseProductDetailEnroll.js

```diff
@@ -168,12 +223,88 @@ export class CourseProductDetailEnroll extends React.Component<
       : null
   }
 
+  cancelEnrollment() {
+    const { upgradeEnrollmentDialogVisibility } = this.state
+
+    this.setState({
+      upgradeEnrollmentDialogVisibility: !upgradeEnrollmentDialogVisibility
+    })
+  }
+
+  renderRunSelectorButtons(run: EnrollmentFlaggedCourseRun) {
+    const { courseRuns } = this.props
+
+    return (
+      <>
+        <label htmlFor="choose-courserun">Choose a date:</label>
+        <select
+          onChange={this.hndSetCourseRun.bind(this)}
+          className="form-control"
+        >
+          {courseRuns &&
+            courseRuns.map((elem: EnrollmentFlaggedCourseRun) => (
+              <option
+                selected={run.id === elem.id}
+                value={elem.id}
+                key={`courserun-selection-${elem.id}`}
+              >
+                {formatPrettyDate(moment(new Date(elem.start_date)))} -{" "}
+                {formatPrettyDate(moment(new Date(elem.end_date)))}
+              </option>
+            ))}
+        </select>
+      </>
+    )
+  }
+
+  getEnrollmentForm(run: EnrollmentFlaggedCourseRun, showNewDesign: boolean) {
+    const csrfToken = getCookie("csrftoken")
+
+    return showNewDesign ? (
+      <form action="/enrollments/" method="post">
+        <input type="hidden" name="csrfmiddlewaretoken" value={csrfToken} />
+        <input type="hidden" name="run" value={run ? run.id : ""} />
+        <button type="submit" className="btn enroll-now enroll-now-free">
+          Or, Enroll for Free without a certificate
```

Comment:
Remove "Or, " -- it just reads strange. 

Can we bold Enroll for Free? 

```suggestion
          <em>Enroll for Free</em> without a certificate
```